CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [Unreleased 5.1.7]
 ## Features
 - AVAILABITY REQUEST : error message if availableStock is less than realQty
+- ACCOUNTING EXPORT : french export of accounting move for admistration mustn't contains the move line on a particular kinds of accounts.
```

Comment:
Change it to meaningful like : Accounting type selection added for reports 'Export journal entry to Administration' and 'Export journal entry to Acc. soft.'